### PR TITLE
Add Maintainer and Governance Info

### LINKED ISSUE #990: Add governance model information
**Problem Statement**
Tern has some information about how to contribute to the project but no information about a path to maintainership or general governance. Although this is a small project in its current state, this information would be nice to have in place as a baseline and it can always be updated later. Pulling in @geekygirldawn to see if she has any helpful resources or templates that might help us get started.

### PR DESCRIPTION
**Pull request by rnjudge**
It's important for open source projects to have clear guidelines and
expectations around governance and maintainership.

Resolves #990

Signed-off-by: Rose Judge <rjudge@vmware.com>

### CODE REVIEW

**Comment by rnjudge** [comment]
@geekygirldawn could you review when you have a moment? :)

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+* **Contributors:** Regular contributions to projects (documentation, code reviews, responding to issues, participation in proposal discussions, contributing code, etc.). 
+* **Maintainers**: The Tern project leaders. They are responsible for the overall health and direction of the project; final reviewers of PRs and responsible for releases. Maintainers are expected to contribute code and documentation, review PRs including ensuring quality of code, triage issues, proactively fix bugs, and perform maintenance tasks for Tern.
+
+### Maintainers
+
+New maintainers must be nominated by an existing maintainer and must be elected by a supermajority of existing maintainers. Likewise, maintainers can be removed by a supermajority of the existing maintainers or can resign by notifying one of the other maintainers.
+
+### Supermajority
+
+A supermajority is defined as two-thirds of members in the group.
```
Comment (nishakm) [inline GOVERNANCE.md:21]: There are currently only two maintainers. How can we get to a supermajority without a third one?

**Review by nishakm — CHANGES_REQUESTED** [review]
As a small project with only 2 maintainers, I don't think a supermajority vote is possible. I would suggest going with lazy consensus on decisions for now.

**Comment by geekygirldawn** [comment]
> As a small project with only 2 maintainers, I don't think a supermajority vote is possible. I would suggest going with lazy consensus on decisions for now.

I agree with the point about a supermajority for now. 

I would leave the **project** decision-making as lazy consensus, but for new maintainers, governance changes, and removing maintainers, I would remove the lazy and require consensus between the 2 of you until we have additional maintainers. People get busy, distracted, and miss things, so I prefer not to have lazy consensus for big decisions. 

Does this seem reasonable?

Otherwise, it looks great, and thank you so much for doing this!

**Comment by nishakm** [comment]
> I would leave the **project** decision-making as lazy consensus, but for new maintainers, governance changes, and removing maintainers, I would remove the lazy and require consensus between the 2 of you until we have additional maintainers. People get busy, distracted, and miss things, so I prefer not to have lazy consensus for big decisions.

+1 on this. Thanks for reviewing @geekygirldawn!

**Comment by rnjudge** [comment]
@geekygirldawn @nishakm changed the "Lazy Consensus" section to be:

```
To maintain velocity in the Tern project, the concept of [Lazy Consensus](http://en.osswiki.info/concepts/lazy_consensus) is practiced. Lazy Consensus is practiced for all projects and decisions in the `tern-tools` org.

Lazy consensus does _not_ apply to the process of:
* Removal of maintainers from Tern
* Governance changes
* Adding new maintainers to Tern

For the above mentioned processes, consensus between @rnjudge and @nishakm is required until there are enough maintainers for a supermajority vote.
```

**Review by nishakm — APPROVED** [review]

**Pull request merged**